base/commandLine.cpp

```diff
@@ -805,6 +815,15 @@ Common::String parseCommandLine(Common::StringMap &settings, int argc, const cha
 				}
 			END_OPTION
 
+			DO_LONG_OPTION("md5-path")
+				Common::FSNode path(option);
+				if (!path.exists()) {
+					usage("Non-existent file path '%s'", option);
+				} else if (!path.isReadable()) {
+					usage("Non-readable file path '%s'", option);
+				}
+			END_OPTION
+
```

Comment:
I am wondering if it is better to have this option, or to use the `--path=<PATH>` option that is already used for the `--add`, `--detect`, and `--auto-detect` commands.

One advantage I can see in having a separate one for the `--md5` command is that it is expecting a file, while the other ones are expecting directories. So it could be confusing to share the same `--path=` option. And that also means we could add a `path.isDirectory()` check here to provide a relevant usage message.

Reply:
I had used --p and --path in a previous commit. But on further testing I ran into the exact same case that you have described here (directories instead of files). Also, the usage errors they throw are ```Non-existent game path``` or ```Non-readable game path```. [link](https://github.com/scummvm/scummvm/blob/9ebd702a4cf1962d492c74301bb2265313fe62d4/base/commandLine.cpp#L706). But here we should use ```Non-existent file path``` instead.

So I used --md5-path instead of --path and squashed my changes to this commit.
I will add the isDirectory() function to alert the user if they enter a directory path